File: CentralAuth/Commons/Models/ProjectToProject.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace CentralAuth.Commons.Models
{
    public class ProjectToProject : MetaClass
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public string ProjekApiName { get; set; }
        public Project Projek { get; set; }
        [Required]
        public string KolaborasiApiName { get; set; }
        public Nullable<bool> Approve { get; set; }
    }
}

File: CentralAuth/ClientApp/src/app/models/project-to-project.ts
import { Project } from './project';

export class ProjectToProject {
    id: number;
    projekApiName: string;
    projek: Project;
    kolaborasiApiName: string;
    kolaborasi: Project;
    approve: boolean|null;
    public constructor(init?: Partial<ProjectToProject>) {
        Object.assign(this, init);
    }
}
